test/cors.test.js

```diff
@@ -200,17 +186,15 @@ test('Should add cors headers (custom values)', t => {
     delete res.headers.date
     t.strictEqual(res.statusCode, 204)
     t.strictEqual(res.payload, '')
-    t.deepEqual({
+    t.match(res.headers, {
       'access-control-allow-origin': 'example.com',
       vary: 'Origin',
       'access-control-allow-credentials': 'true',
       'access-control-expose-headers': 'foo, bar',
       'access-control-allow-methods': 'GET',
       'access-control-allow-headers': 'baz, woo',
-      'access-control-max-age': '123',
-      'content-length': '0',
```

Comment:
This content-length should remain since it is added by 
https://github.com/fastify/fastify-cors/blob/0365bdbe79ce03227360df093d4d6c596e506fc3/index.js#L85-L87